src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -676,3 +683,33 @@ export const get4142Selection = (disabilities) => {
     return false;
   }, false);
 };
+
+export const AddressDescription = () => (
+  <div>
+    <p>
+      This is the contact information we have on file for you. We’ll send any important
+      information about your disability claim to the address listed here. Any updates
+      you make here to your contact information will only apply to this application.
+    </p>
+    <p>
+      If you want to update your contact information for all your VA accounts, please go
+      to your profile page.
+    </p>
+    <p>
+      <a href="/profile">Go to my profile</a>.
+    </p>
+  </div>
+);
+
+export const PaymentDescription = () => (
+  <div>
+    <p>
+    This is the bank account information we have on file for you. We’ll pay your
```

Comment:
It looks like the spacing/indentation is slightly different here.

Reply:
bah! Thanks!